### DISCUSSION TITLE
How to report bugs? چگونه باگ را گزارش کنیم؟

### DISCUSSION THREAD

**Discussion by hiddify-com** [hiddify/Hiddify-Manager#141] [Bug باگ: please use issue]
لطفا صرفا از طریق [issues ](https://github.com/hiddify/hiddify-config/issues)باگ یا مشکلات را گزارش کنید

**Comment by Lenjerman2022** [comment]
بچه های قدرتمند هیدیفای ایا پنل شما با دیتاسنتر دیجیتال اوشن با ایپی هاش مشکل داره الان یک هفتس رو هیچ ایپی نصب نمیشه نصبم میشه بالا نمیاد مشکلی هست توضیح بدید بچه های کادر تلگرامتون هم همه عصبی هستند
